src/deluge/model/mod_controllable/mod_controllable_audio.cpp

```diff
@@ -1842,11 +1845,11 @@ void ModControllableAudio::receivedCCFromMidiFollow(ModelStack* modelStack, Clip
 								// performance view
 								bool editingParamInAutomationOrPerformanceView = false;
 								RootUI* rootUI = getRootUI();
-								if (rootUI == &automationClipView || rootUI == &performanceSessionView) {
+								if (rootUI == &automationView || rootUI == &performanceSessionView) {
```

Comment:
should this be current UI? no need to refresh if they aren't the top layer

Reply:
> should this be current UI? no need to refresh if they aren't the top layer

For Automation View yes, because you can still see the automation view grid and if you're controlling the param currently shown on that grid, the grid refreshes even while you're in the menu

When I eventually add the regular sound menu to automation view, it will be nice to see the greyed out grid refresh as well when you're editing param values in the menu